pom.xml

```diff
@@ -55,32 +55,32 @@
     <netty.version>4.1.52.Final</netty.version>
 
     <!-- It's easy for Jackson dependencies to get misaligned, so we manage it ourselves. -->
-    <jackson.version>2.11.2</jackson.version>
+    <jackson.version>2.11.3</jackson.version>
 
     <cassandra-driver-core.version>3.10.2</cassandra-driver-core.version>
-    <jooq.version>3.13.4</jooq.version>
+    <jooq.version>3.13.5</jooq.version>
     <micrometer.version>1.5.5</micrometer.version>
     <spring-boot.version>2.3.4.RELEASE</spring-boot.version>
     <!-- MySQL connector is GPL, even if it has an OSS exception.
          https://www.mysql.com/about/legal/licensing/foss-exception/
 
          MariaDB has a friendlier license, LGPL, which is less scary in audits.
     -->
-    <mariadb-java-client.version>2.6.2</mariadb-java-client.version>
+    <mariadb-java-client.version>2.7.0</mariadb-java-client.version>
     <HikariCP.version>3.4.5</HikariCP.version>
     <slf4j.version>1.7.30</slf4j.version>
     <auto-value.version>1.7.4</auto-value.version>
     <git-commit-id.version>4.0.2</git-commit-id.version>
 
     <!-- Test only dependencies -->
-    <junit.version>4.13</junit.version>
+    <junit.version>4.13.1</junit.version>
```

Comment:
this applies the o.. soo.. important.. junit update that dependabot decided to nag the entire java ecosystem about